### ISSUE TITLE
Triggering bot update?

### ISSUE THREAD

**Issue by Fdawgs** [conda-forge/poppler-feedstock#126]
### Comment:

Hi there,

How do we go about triggering the bot to make an update PR?
Poppler is currently on 22.04, whilst this repo is still on 22.01.

**Comment by xylar** [comment]
@Fdawgs, I don't think the bot can notice changes here because of the URL from which the code comes.  Updated have to be done manually.

**Comment by Fdawgs** [comment]
Thanks @xylar, and apologies I had assumed it was the bot who opened PRs for new updates from looking at previous PRs

**Comment by xylar** [comment]
@Fdawgs, you're right.  The last update was done manually but the bot was working fine before that. 

I also don't see any sign of trouble here:
https://conda-forge.org/status/
under Version Updates and Errored.  So I think the bot just isn't seeing the updates anymore for some reason.

**Comment by Fdawgs** [comment]
Sorry for digging this back up (can open a separate issue if wanted), is there a way to kick the bot back into action?

**Comment by xylar** [comment]
@Fdawgs, this might be a place to ask for help: https://gitter.im/conda-forge/regro-cf-autotick-bot
